### ISSUE TITLE
Document config.json when using reverse proxy

### ISSUE THREAD

**Issue by jernst** [solid/node-solid-server#819]
I assume this is the intended setup?
* If I run Apache on port 80 + 443
* and I run sold-server at port 5001
* and my hostname is `example.com`
then the `config.json` file has those entries:
```
{
  "port": "5001",
  "serverUri": "https://example.com",
}
```
Is that right?

**Comment by kjetilk** [comment]
Yes, that looks about right. I have a slightly different setup where I have HAProxy on a different machine than where I have solid-server, but it is essentially the same thing.

**Comment by melvincarvalho** [comment]
Some notes in the wiki

https://github.com/solid/node-solid-server/wiki/Running-Solid-behind-a-reverse-proxy

**Comment by jernst** [comment]
I saw those notes on the wiki, but the core of my question -- about the config file -- isn't addressed there. Specifically whether the port also needs to be on `serverUri`.

**Comment by kjetilk** [comment]
@jernst `serverUri` is what the world should see, so if you're proxying it, then it shouldn't have the port.
